source/common/http/async_client_impl.h

```diff
@@ -274,6 +274,7 @@ class AsyncStreamImpl : public AsyncClient::Stream,
     const Router::VirtualHost& virtualHost() const override { return virtual_host_; }
     bool autoHostRewrite() const override { return false; }
     bool includeVirtualHostRateLimits() const override { return true; }
+    Router::VhRateLimitOptions virtualRateLimitsOptions() const override { return vh_rate_limits_; }
```

Comment:
Maybe the name should be more explicit, e.g. virtualHostRateLimitOptions. note the 'host'